plugins/scroll-options/src/utils.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Gets the current location of the workspace when there's no drag surface.
+ * @param {!Blockly.WorkspaceSvg} ws The workspace to calculate.
+ * @return {!Blockly.utils.Coordinate} The current workspace coordinate.
+ */
+export const getTranslation = (ws) => {
+  const translation = ws.svgBlockCanvas_.getAttribute('transform');
```

Comment:
How about moving the check for `dragSurface` into this function and having it automatically return either `dragSurface.getWsTranslation` or the rest of this function? That lets you avoid duplicating code in the three places where this is called.